app/controllers/api/v1/soap_api_controller.rb

```diff
@@ -0,0 +1,53 @@
+class Api::V1::SoapApiController < Api::V1::BaseController
+  soap_service namespace: 'urn:WashOut', wsse_auth_callback: ->(email, password) {
+    user = User.find_by(email: email)
+    return false unless user.present?
+    return user.valid_password?(password)
+  }
+
+  soap_action :get_final_cites_certificate,
+              args: {
+                CertificateNumber: :string,
+                TokenId: :string,
+                IsoCountryCode: :string
+              },
+              return: :string
+  def get_final_cites_certificate
+    render xml: Adapters::SimpleAdapter.run(Adapter.first).to_xml
```

Comment:
Would be good to look up the adapter using the iso code provided. I appreciate we only have one adapter for now :D  but still it's not good to depend on the order of adapters to get this to work. The logic to look up the adapter could be similar as in here: https://github.com/unepwcmc/epix/blob/master/app/controllers/permits_controller.rb#L38